Plugin/src/SofaPython3/PrefabParameter/DataAlias.cpp

```diff
@@ -0,0 +1,69 @@
+#include "DataAlias.h"
+
+namespace sofa::core::objectmodel
+{
+
+
+DataAlias::DataAlias(BaseData *parent, BaseData* link)
```

Comment:
I think there is a component called DataAlias in Sofa, maybe we can find a different name. In addition to me this is not an alias (a different naming) but more a kind of "lazy" data value fetching.